bin/cli.js

```diff
@@ -16,11 +17,22 @@ argv
     flags[flagKey] = keyValue[1] || true;
   });
 
+let url;
```

Comment:
I think in future we can replace all checks here by some validator. It will make cli file more clear.